experimental/design-ios-gradle-dsl/build.gradle.kts

```diff
@@ -0,0 +1,65 @@
+plugins {
+    kotlin("multiplatform")
+    id("org.jetbrains.compose")
+}
+
+kotlin {
+    iosX64()
+    iosArm64()
+}
+
+compose.ios {
+    application {
+        mainClass = "Main_ios.kt"
```

Comment:
Entry point maybe? What will be going on under the hood?

Reply:
I don't have a strong opinion on this one. On one hand, JVM desktop uses `mainClass`, which clearly represents, that a bytecode class name is expected. On the other hand, Kotlin/Native target's configurations use `entryPoint`

Reply:
I just copied this from desktop config.
Yeah, I think entryPoint is better!